src/pe_reports/data/db_query.py

```diff
@@ -64,6 +62,20 @@ def get_orgs(conn):
             close(conn)
 
 
+def get_orgs_df():
+    """Query organizations table for new orgs."""
```

Comment:
Originally it was new orgs but it has been changed since. This is addressed in this commit : [bbfa6b8](https://github.com/cisagov/pe-reports/pull/282/commits/bbfa6b86259e0a8a8f71f7d2eca294533bda38bf)